### PR TITLE
[plugin.video.lbry] v0.1.2

### PR DESCRIPTION
**Pull request by accumulator**
### Description
New plugin for accessing LBRY. 
Plugin needs the LBRY client to function. The client doesn't have to be installed on the same host as Kodi.
Initial functionality:
- Search 
- Play video
- Follow channels
- add to and browse Watch Later playlist

### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.lbry/addon.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="lbry.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">Plugin for LBRY</summary>
+        <description lang="en_GB">LBRY is a secure, open, and community-run digital marketplace</description>
+        <language>en_GB</language>
+        <platform>all</platform>
+        <license>GPL2</license>
```
Comment (enen92) [inline plugin.video.lbry/addon.xml]: GPL-2.0-only or GPL-2.0-or-later (spdx identifier)

File: plugin.video.lbry/addon.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        <import addon="script.module.routing" version="0.2.3"/>
+        <import addon="script.module.requests" version="2.22.0"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="lbry.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">Plugin for LBRY</summary>
+        <description lang="en_GB">LBRY is a secure, open, and community-run digital marketplace</description>
+        <language>en_GB</language>
```
Comment (enen92) [inline plugin.video.lbry/addon.xml]: please remove the language tag, this applies to content only. If you want to limit this to english, please use `en`

File: plugin.video.lbry/resources/lib/plugin.py [hunk 3]
```diff
@@ -0,0 +1,405 @@
...
+            menu.append((
+                tr(30207) % ch_name, 'Container.Update(%s)' % plugin.url_for(lbry_channel, uri=serialize_uri(item['signing_channel']),page=1)
+            ))
+        menu.append((
+            tr(30205) % ch_name, 'RunPlugin(%s)' % plugin.url_for(plugin_follow, uri=serialize_uri(item['signing_channel']))
+        ))
+
+    if repost != None:
+        if 'signing_channel' in repost:
+            plot = ('[COLOR yellow]Repost in %s[/COLOR]\n' % repost['signing_channel']['name']) + plot
```
Comment (enen92) [inline plugin.video.lbry/resources/lib/plugin.py]: this needs to be localized

**Review by enen92 — CHANGES_REQUESTED** [review]
See comments.
Thanks for the submission. A version for matrix would be nice :)

**Comment by accumulator** [comment]
> See comments.

All fixed.

> Thanks for the submission. A version for matrix would be nice :)

No problem :)
Plugin should be python3 compatible. Is there anything else currently blocking matrix compatibility?

**Comment by enen92** [comment]
Yes, the xbmc.python version in addon.xml has to be bumped to 3.0.0 and the addon submitted to the matrix branch of this repo for it to be available in matrix.

**Review by enen92 — APPROVED** [review]

**Comment by accumulator** [comment]
> Yes, the xbmc.python version in addon.xml has to be bumped to 3.0.0 and the addon submitted to the matrix branch of this repo for it to be available in matrix.

Ok. I can make a PR but currently have no Matrix version to test against..

**Comment by enen92** [comment]
I'd prefer you handle that new PR when you actually have tested the addon under matrix :)
You can use portable mode (ref https://kodi.wiki/view/Portable_mode) to keep two version of kodi on the same machine.

**Pull request merged**